### PR TITLE
Feature mes 4287 cat c field display

### PR DESCRIPTION
**Pull request by rossfauldsbjss**
MES-4287
Make changes for Cat C data fields. Added a banner which was in the design but not in the page.
Disabled validation on vehicle check questions to make it easier to test during development of Cat C
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/pages/pass-finalisation/cat-c/_tests_/pass-finalisation.cat-be.page.spec.ts [hunk 1]
```diff
@@ -0,0 +1,180 @@
+import { ComponentFixture, async, TestBed, fakeAsync, tick } from '@angular/core/testing';
```
Comment (MarkPatt89) [inline src/pages/pass-finalisation/cat-c/_tests_/pass-finalisation.cat-be.page.spec.ts]: filename references `cat-be`

File: src/pages/pass-finalisation/cat-c/_tests_/pass-finalisation.cat-be.page.spec.ts [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+describe('PassFinalisationCatCPage', () => {
+  let fixture: ComponentFixture<PassFinalisationCatCPage>;
+  let component: PassFinalisationCatCPage;
+  let store$: Store<StoreModel>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        PassFinalisationCatCPage,
+        MockComponent(PracticeModeBanner),
```
Comment (MarkPatt89) [inline src/pages/pass-finalisation/cat-c/_tests_/pass-finalisation.cat-be.page.spec.ts]: No need for practice banner mock

File: src/pages/pass-finalisation/cat-c/_tests_/pass-finalisation.cat-be.page.spec.ts (hunk 1 above)
Comment (rossfauldsbjss) [reply src/pages/pass-finalisation/cat-c/_tests_/pass-finalisation.cat-be.page.spec.ts]: I didn't introduce this in this PR. but a good spot. Guess I'll fix it

File: src/pages/pass-finalisation/cat-c/_tests_/pass-finalisation.cat-be.page.spec.ts (hunk 2 above)
Comment (rossfauldsbjss) [reply src/pages/pass-finalisation/cat-c/_tests_/pass-finalisation.cat-be.page.spec.ts]: will remove

**Review by MarkPatt89 — APPROVED** [review]
Once the file name is changed, all will be well.

File: src/pages/pass-finalisation/cat-c/_tests_/pass-finalisation.cat-be.page.spec.ts (hunk 1 above)
Comment (rossfauldsbjss) [reply src/pages/pass-finalisation/cat-c/_tests_/pass-finalisation.cat-be.page.spec.ts]: Will remove this file entirely there is a cat c spec file already. suspect it was brought in as a result of merging with develop

File: src/pages/pass-finalisation/cat-c/_tests_/pass-finalisation.cat-be.page.spec.ts (hunk 2 above)
Comment (rossfauldsbjss) [reply src/pages/pass-finalisation/cat-c/_tests_/pass-finalisation.cat-be.page.spec.ts]: Will remove this file entirely there is a cat c spec file already. suspect it was brought in as a result of merging with develop

**Review by szabikr — APPROVED** [review]
looks good

**Pull request merged**